src/QA/QA.Web/ViewModels/ManageProvider/BlockedOrganisationsViewModel.cs

```diff
@@ -6,5 +6,7 @@ public class BlockedOrganisationsViewModel
     {
         public List<BlockedOrganisationViewModel> BlockedOrganisations { get; set; } = new List<BlockedOrganisationViewModel>();
         public bool HasBlockedOrganisations => BlockedOrganisations.Count > 0;
+
+        public bool HasOneBlockedProviders => BlockedOrganisations.Count == 1;
```

Comment:
Should this be called `HasOneBlockedOrganisation` to be in keeping with the rest of the class.